obs-service-replace_using_package_version.spec

```diff
@@ -31,6 +31,7 @@ Source2:        LICENSE
 BuildRequires:  sed
 Requires:       python3-docopt
 Requires:       python3-setuptools
```

Comment:
I am not sure this is needed any longer with such a change. IIRC this was only included for the `pkg_resource` import at runtime.

Reply:
Let's remove it, it shouldn't be needed.